profile/arc_datamap_ro_crate.md

```diff
@@ -99,10 +100,13 @@ It is based on [schema.org/PropertyValue](https://schema.org/PropertyValue) and
 |----------|----------|---------------|-------------|
 |@type |MUST|Text|must be '[schema.org/PropertyValue](https://schema.org/PropertyValue)'|
```

Comment:
Some of the `Must` in the description are starting with lowercase `m`. Please put to upper like in the lines below.